openassessment/xblock/staff_area_mixin.py

```diff
@@ -460,7 +460,37 @@ def clear_student_state(self, user_id, course_id, item_id, requesting_user_id):
                 item_id,
                 clear_state=True
             )
-            # TODO: try to remove the above pylint disable once edx-submissions release is done
+
+    def clear_team_state(self, user_id, course_id, item_id, requesting_user_id):
+        error_msg_base = 'Attempted to clear team state for anonymous user {} '.format(user_id)
+        try:
+            user_team = self.get_team_for_anonymous_user(user_id)
+        except ObjectDoesNotExist:
+            logger.warning(error_msg_base + 'but was unable to resolve to a real user')
+            return
+
+        if user_team is None:
+            logger.warning(
+                error_msg_base + 'but they are not on a team for course {} item {}.'.format(
+                    course_id, item_id
+            ))
+            return
+
+        from submissions import api as submission_api
+        from submissions import team_api as team_submissions_api
+
+        try:
+            team_submission = team_submissions_api.get_team_submission_for_team(course_id, item_id, user_team.team_id)
+        except TeamSubmissionNotFoundError:
+            logger.warning(error_msg_base + "course {} item {} but no team submission was found for team {}".format(
+                course_id, item_id, user_team.team_id
+            ))
+
+        # Remove the submission from grading pool
+        self._cancel_team_workflow(team_submission['team_submission_uuid'], "Student and team state cleared", requesting_user_id)
```

Comment:
Are these comments read/parsed anywhere? If so, should this be a defined constant or are we okay to use this ad-hoc like this?

Reply:
It looks like this is not parsed anywhere? I couldn't find any references in code.